mobly/controllers/attenuator_lib/minicircuits.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3.4
+#
+# Copyright 2016 Google Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module has the class for controlling Mini-Circuits RCDAT series
+attenuators over Telnet.
+
+See http://www.minicircuits.com/softwaredownload/Prog_Manual-6-Programmable_Attenuator.pdf
+"""
+
+from mobly.controllers import attenuator
+from mobly.controllers.attenuator_lib import telnet_client
+
+
+class AttenuatorDevice(object):
+    """This provides a specific telnet-controlled implementation of
+    AttenuatorDevice for Mini-Circuits RC-DAT attenuators.
+
+    Attributes:
+        path_count: The number of signal attenuation path this device has.
+        _telnet_client: The telnet client used to communicate with the
```

Comment:
I don't think private attributes are typically documented in the user-facing module documentation. If this comment is valuable, it should probably be a comment at the variable declaration site.